Sources_v3/StreamChatUI/UIConfig.swift

```diff
@@ -209,6 +211,7 @@ public extension UIConfig {
             ChatMessageOnlyVisibleForCurrentUserIndicator.self
         public var threadArrowView: ChatMessageThreadArrowView<ExtraData>.Type = ChatMessageThreadArrowView<ExtraData>.self
         public var threadInfoView: ChatMessageThreadInfoView<ExtraData>.Type = ChatMessageThreadInfoView<ExtraData>.self
+        public var errorIndicatorIcon = UIImage(named: "error_indicator", in: .streamChatUI)!
```

Comment:
I don't think it's needed to expose every icon here. If the button was a custom type (as it should be), you should be able to change the icon via `defaultAppearance`

Reply:
removed, set icon in `defaultAppearance` in 1f84cffdc3bf89c97e082e340c723b2483bc91e4